### PR TITLE
Add netcdf conversion to preindustrial configuration

### PR DESCRIPTION
**Pull request by blimlim**
This pull request adds the um2nc based netcdf conversion as a postscript to the preindustrial configuration. Fields file deletion is switched on by default.

### CODE REVIEW

File: scripts/NetCDF-conversion/UM_conversion_job.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#PBS -l ncpus=1
+#PBS -l mem=20GB
+#PBS -l jobfs=0GB
+#PBS -q normal
+#PBS -l walltime=00:40:00
+#PBS -l wd
+
+module use /g/data/vk83/modules
+module load payu
+
```
Comment (aidanheerdegen) [inline scripts/NetCDF-conversion/UM_conversion_job.sh:11]:
```suggestion

# Command below calls https://github.com/ACCESS-NRI/um2nc-standalone/blob/main/umpost/conversion_driver_esm1p5.py
# By default UM atmosphere fields files are deleted after conversion to save space. 
# Remove --delete-ff command line option to retain original files for testing purposes
```

**Review by aidanheerdegen — APPROVED** [review]
Just a suggestion to add a comment in the script. It can be very hard to find where the original code lives, so a pointer here could save someone a lot of time.

**Review by aidanheerdegen — APPROVED** [review]
Looks great. Thanks!

File: scripts/NetCDF-conversion/UM_conversion_job.sh (hunk 1 above)
Comment (blimlim) [reply scripts/NetCDF-conversion/UM_conversion_job.sh:11]: Great idea!

**Pull request merged**